modules/swagger-codegen/src/main/resources/python/README.mustache

```diff
@@ -128,3 +128,28 @@ Class | Method | HTTP request | Description
 {{/isOAuth}}
 
 {{/authMethods}}
+
+## Integration with Google Cloud Platform
+
+Swagger client is using urllib3 as default http library, however Google Cloud Platform disallow it. 
+There are two ways you can use urllib3 in Appengine Standard: patch it to use App Engine’s URLFetch or enable sockets. 
+Which one to use depends on your situation and needs. URLFetch is more cost-effective as long as your usage is within the limitations for free app. Sockets are only available for paid apps and are in beta at the time of writing.
+
+The first solution:
+
+```
+import swagger_client
+from urllib3.contrib.appengine import AppEngineManager
+
+# create swagger client
+client = swagger_client.PetApi()
```

Comment:
nit: I suggest adapting this example to Smooch's context

Reply:
Just replaced `swagger_client.PetApi()` with `smooch.AppApi()`.